WearSpeakerSample/README.md

```diff
@@ -1,11 +1,16 @@
 
-Android WearSpeakerSample Sample
+WearSpeakerSample
 ================================
 
 A sample that shows how you can record voice using the microphone on a wearable and
 play the recorded voice or an mp3 file, if the wearable device is connected to a speaker
 (bluetooth or built-in).
 
+This sample is also written entirely with Jetpack Compose, using [Wear-specific components provided
+by AndroidX](https://developer.android.com/jetpack/androidx/releases/wear-compose).
+The Compose UI is fairly advanced, using [ConstraintLayout for Compose](https://developer.android.com/jetpack/compose/layouts/constraintlayout).
```

Comment:
Actually, can you also call out this uses several effects APIs and link to the guide? That's pretty important too.

Reply:
Done!